### ISSUE TITLE
Clicking Abort when there is a problem loading a .csproj file should not exit the app

### ISSUE THREAD

**Issue by jnm2** [paul1956/CSharpToVB#8]
I want to leave the app open and try again after changing things on disk.

It's not clear that 'Retry' or 'Ignore' mean "stop opening the project and let me try again."

**Comment by jnm2** [comment]
This is actually worse than I reported. If you detach the debugger (to stop hitting `Stop` so many times) or you run without debugging, the app simply crashes to the desktop rather than presenting an error message and a choice.

**Comment by paul1956** [comment]
Abort usually means exit the app, the other 2 options let you make changes and don't exit. But in the case of MSBuild failures not sure what you could do. I have updated the title to explain abort.

**Comment by jnm2** [comment]
In the case of MSBuild failures, couldn't the app stay open and be put in the same state as before the user clicked 'Open project'?

**Comment by paul1956** [comment]
I need to look at this, trying to convert Project files was probably something I should not have tried.

**Comment by paul1956** [comment]
This issue is fixed, why I can't open a project with MSBuild anymore is a mystery.
